TESTING.md

```diff
@@ -532,6 +533,29 @@ Look at `TestResourceInfoProviders` to see a full example of how to use the meth
 once in the HCL script. If it is not, the variable `testAccProvider` may not get initialised, and if that happens,
 test checks that use the expression `conn := testAccProvider.Meta().(*VCDClient)` will panic.
 
+## Partitioned tests
+
+We can run tests in a partitioned way, by splitting the tests across several VCDs (nodes) each of which will run a given portion of the tests.
+
+To activate this modality, we use the following options:
+
+* `-vcd-partition=N` indicates the number of partitions running the test suite.
+* `-vcd-partition-node=N` indicates which node the current VCD will run.
+* `-vcd-partition-tests-file=FileName` (optional) provides the list of tests to run
+
+When partition mode is enabled, the current node will skip all the tests that are not assigned to it.
+
+The test assignment, by default, happens by getting the list of all tests, sorting them alphabetically, and then assigning them to each node in sequence. 
+If a tests file name was provided, the node will just use such file as its assignment.
+
+Each node produces several files:
+
+* `{BUILD_NUMBER}-tests-retrieved-in-node-{NODE_NUMBER}-{VCD_VERSION}.txt` All the tests found in the current node
+* `{BUILD_NUMBER}-tests-planned-in-node-{NODE_NUMBER}-{VCD_VERSION}.txt` The tests that will run in the current node
+* `{BUILD_NUMBER}-tests-processed-in-node-{NODE_NUMBER}-{VCD_VERSION}.txt` The tests that have been processed in this node
+* `{BUILD_NUMBER}-out-{NODE_NUMBER}.txt` A sentinel file signifying that the tests in the current node are finished. It contains the error code of the run (0=success, 1=failure)
```

Comment:
We could probably match these names for `.gitignore` to avoid accidental merge

Reply:
Done